search/src/main/scala/weco/api/search/services/ImagesRequestBuilder.scala

```diff
@@ -112,12 +111,15 @@ class ImagesRequestBuilder(queryConfig: QueryConfig)
         )
       case GenreFilter(genreLabels) =>
         termsQuery("query.source.genres.label.keyword", genreLabels)
+      case GenreConceptFilter(conceptIds) =>
+        if (conceptIds.isEmpty) NoopQuery
+        else termsQuery("query.source.genres.concepts.id", conceptIds)
```

Comment:
Do we need a special case for no concept IDs passed for genre filters only, or do you think we should be doing this for all filters?

Reply:
I think it should be for all filters.  The current behaviour for other filters is that an empty filter causes it not to match anything, which I discovered when adding the Cardinality test for this.

I didn't want to extend it as part of this work, as it was already big enough.
I'll add a ticket to extend this behaviour.